services/QuillLMS/client/app/bundles/Connect/components/questions/questions.jsx

```diff
@@ -115,9 +115,6 @@ class Questions extends React.Component {
           gradeIndex: `unmatched${response.questionUID}`,
         };
         sleep(150);
-        this.props.dispatch(
```

Comment:
It seems wrong to just delete these dispatches, but their function values are undefined, so it should be fine.

Reply:
Whoa, I wonder how that happened.